drivers/pinctrl/intel/pinctrl-intel.c

```diff
@@ -1292,6 +1324,17 @@ static int intel_pinctrl_pm_init(struct intel_pinctrl *pctrl)
 			return -ENOMEM;
 
 		communities[i].intmask = intmask;
+
+		if (dmi_first_match(dmi_retain_hostown_table)) {
+			pctrl->quirks |= PINCTRL_QUIRK_KEEP_HOSTOWN;
```

Comment:
When setting `PINCTRL_QUIRK_KEEP_HOSTOWN` lets also and a log message at info level so we know this quirk is active in case it causes any unexpected problems. Something like "enabling KEEP_HOSTSWOWN quirk on this hw" should suffice.